appveyor.yml

```diff
@@ -34,10 +31,7 @@ before_test:
   - ps: . .\appveyor-setup.ps1
 
 test_script:
-  - 'IF "%COVERAGE%" == "true" (npm run coveralls) ELSE (npm test)'
-
-notifications:
-  - provider: Webhook
-    url: https://coveralls.io/webhook?repo_token=751SiA6QhY5EH4P2ygq7JFWulqiD2ETXO
-    on_build_success: true
-    on_build_failure: true
+  - 'IF "%COVERAGE%" == "true" (npm run cover) ELSE (npm test)'
+  
+after_test:
+  - ps: (Invoke-WebRequest https://codecov.io/bash).toString() | bash -f .\coverage\lcov.info
```

Comment:
Seems it's interpreting the coverage file as bash

![image](https://cloud.githubusercontent.com/assets/1210224/18246389/b5690ec0-736b-11e6-8624-e52cb9c176f7.png)